pkg/options/network/create.go

```diff
@@ -11,7 +11,12 @@ const (
 	short   = "Network aws cloudformation stack"
```

Comment:
@den-rgb Personally, these feel a bit too literal. From a customer perspective, the role of this command is to quickly create your network configuration for your HCP clusters. As a suggestion (unless this has already been given the +1 elsewhere)

- short: Create required network configuration for ROSA HCP clusters
- long: Quickly create the required network configuration in AWS to support deployment of ROSA HCP clusters
- example: Create the required network configuration for a publicly accessible, ROSA HCP cluster

I also find the inclusion of the `to delete the AWS Cloudformation stack` here a bit confusing as it is an implementation detail and feels more appropriate to a log statement after the stack has been created.

Reply:
It was QE that asked for a more literal example as well as the generic one, we also have to remember this functionality is aimed at new users
As discussed with the team previously we decided we didnt want the delete command after cluster creation and it should be included in the help instead.

Reply:
Also this command isn't only going to be used for HCP related resources it can be used for anything, so it wouldn't make sense to have the long and the short as above